### PR TITLE
Remove Redis Requirement

### PR DESCRIPTION
**Pull request by michaeljs1990**
I don't feel like you should have to install redis to use this library.
This makes it so that you can use the library without having fatal errors
thrown due to a missing optional dependency like redis.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
         }
     ],
     "require": {
-        "php": ">=5.6.3",
+        "php": ">=5.5",
```
Comment (michaeljs1990) [inline composer.json]: I may be missing something but the code here seems to run on php 5.5 just fine as well as all the dependencies running on php 5.5. Running the latest and greatest is cool but no need to require it in this case

**Comment by michaeljs1990** [comment]
Wanted to use this without having to install php5-redis as it does show as an optional dependency which is not actually the case.

**Comment by bracki** [comment]
While I agree that using the library without `php5-redis` is desirable, I don't see any point in adding changes for EOLed PHP version.
Please remove the other two commits.

**Comment by michaeljs1990** [comment]
Sounds good to me i'll clean it up, thanks for taking a look.

**Comment by bracki** [comment]
Closing due to inactivity.

**Pull request closed without merging**